### ISSUE TITLE
Support unicode values for mapping to punycode BNS names in recipient field of send form

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/blockstack-app#2872]
For added usability (e.g. users can enter `🌎✌🏻.btc` instead of `xn--7bix418m.btc` to send STX to someone).

It'd be prudent to display some sort of guidance about how we're doing this mapping on the fly, perhaps by showing the converted punycode value, even though there are no non-mapped unicode registrations on chain (but could potentially exist in the future with BNS protocol upgrades).

<img width="380" alt="image" src="https://user-images.githubusercontent.com/28991/202448498-a32a4ef8-96c5-4ad0-ae12-73fe6daebe05.png">

**Comment by 384007** [comment]
hi mark!  Thanks to the community for having you. Emoji is the best way to express human beings. punycode is the backend of uni. It is the consensus of the Internet. Strongly support this function

**Comment by sevensato** [comment]
Yes, do it mark! People all ages across globe use emojis in their everyday life. Punycode is good, this is gonna be a big moment for us to build. Thanks team!

**Comment by 314159265359879** [comment]
For clarification for now this functionality is not added. Because of the vulnerability discussed in the issue here: https://github.com/hirosystems/stacks-wallet-web/issues/2962#issuecomment-1369685802

For now you will see:
![image](https://user-images.githubusercontent.com/33360391/224674671-89010b1d-3af5-4654-832b-7a12a2be6062.png)

And type the bns name instead to resolve the STX address (or BTC address, if available). 
![image](https://user-images.githubusercontent.com/33360391/224674935-1cd45007-af02-4f97-8aae-ec4c504119ea.png)
